nixos/server/default.nix

```diff
@@ -24,6 +24,12 @@
     stub-ld.enable = lib.mkDefault false;
   };
 
+  # Restrict the number of boot entries to prevent full /boot partition.
+  #
+  # Servers don't need too many generations.
+  boot.loader.grub.configurationLimit = lib.mkDefault 5;
+  boot.loader.systemd-boot.configurationLimit = lib.mkDefault 5;
```

Comment:
Does this also contain the currently booted kernel?

Reply:
I don't think so

Reply:
I got told the booted kernel will be still protected by /run/booted-system

Reply:
Ok then good to go?